lib/lib.scriptHost.d.ts

```diff
@@ -1,294 +0,0 @@
-/*! *****************************************************************************

```

Comment:
not sure why this appears as deleted. it is still there in master: https://github.com/Microsoft/TypeScript/blob/master/lib/lib.scripthost.d.ts